email_pin/src/lang/en.json

```diff
@@ -61,7 +61,7 @@
   },
   "page_enter_recovery_key": {
     "title": "Enter Recovery Key",
-    "description": "You can also upload the Recovery File which was generated when you otiginally set up your PIN.",
+    "description": "You can also upload the Recovery File which was generated when you originally set up your PIN.",
```

Comment:
I would use 'initially' instead of 'originally' or better this:
`"description": "You can also upload the Recovery File which was generated the first time you set up your PIN."`